include/sdf/Material.hh

```diff
@@ -59,37 +59,45 @@ namespace sdf
     /// specified by a set of three numbers representing red/green/blue,
     /// each in the range of [0,1].
     /// \return Ambient color.
-    public: ignition::math::Color Ambient() const;
+    public: gz::math::Color Ambient() const;
 
     /// \brief Set the ambient color. The ambient color is
     /// specified by a set of three numbers representing red/green/blue,
     /// each in the range of [0,1].
     /// \param[in] _color Ambient color.
-    public: void SetAmbient(const ignition::math::Color &_color);
+    public: void SetAmbient(const gz::math::Color &_color);
 
     /// \brief Get the diffuse color. The diffuse color is
     /// specified by a set of three numbers representing red/green/blue,
     /// each in the range of [0,1].
     /// \return Diffuse color.
-    public: ignition::math::Color Diffuse() const;
+    public: gz::math::Color Diffuse() const;
 
     /// \brief Set the diffuse color. The diffuse color is
     /// specified by a set of three numbers representing red/green/blue,
     /// each in the range of [0,1].
     /// \param[in] _color Diffuse color.
-    public: void SetDiffuse(const ignition::math::Color &_color);
+    public: void SetDiffuse(const gz::math::Color &_color);
 
     /// \brief Get the specular color. The specular color is
     /// specified by a set of three numbers representing red/green/blue,
     /// each in the range of [0,1].
     /// \return Specular color.
-    public: ignition::math::Color Specular() const;
+    public: gz::math::Color Specular() const;
 
     /// \brief Set the specular color. The specular color is
     /// specified by a set of three numbers representing red/green/blue,
     /// each in the range of [0,1].
     /// \param[in] _color Specular color.
-    public: void SetSpecular(const ignition::math::Color &_color);
+    public: void SetSpecular(const gz::math::Color &_color);
+
+    /// \brief Get the specular exponent.
+    /// \return Specular exponent.
+    public: double Shininess() const;
+
+    /// \brief Set the specular exponent.
+    /// \param[in] _shininess Specular exponent.
+    public: void SetShininess(const double _shininess);
```

Comment:
Now these are duplicated, see the lines below

Reply:
My bad! Force-pushed and fixed! https://github.com/gazebosim/sdformat/compare/ff7d50728bf65bc2385580a7d5a02d7cdfaacde4..731ee8b4de53c1e89310f3bad80534893a89b43b